resources/public/api/conf/2.0/application.yaml

```diff
@@ -1181,14 +1237,16 @@ components:
           description: The description of the error.
     AcceptHeaderInvalid:
       title: AcceptHeaderInvalid
-      description: The `Accept` header was not set to `application/vnd.hmrc.2.0+json`.
+      description: The `Accept` header was not set to `application/vnd.hmrc.2.0+json`, `application/vnd.hmrc.2.0+json+xml` or `application/vnd.hmrc.2.0+json-xml`.

```

Comment:
Was there not some Platform problem with writing `json+xml` in API source code and API definitions? That's why we went with `json-xml`.

Reply:
Yeah. We left it in the code in case platform allowed it at some point -- so it's not entirely obvious that's the case just by looking at it.

@akila-sankar Can you remove `application/vnd.hmrc.2.0+json+xml` from the list please?

Reply:
Yes agree @drnaylor-oc will remove from the list